cmd/cmd.go

```diff
@@ -4,16 +4,25 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/ace-teknologi/go-abn/abr"
 	"github.com/spf13/cobra"
 )
 
 const (
-	guidFlagName = "GUID"
+	guidFlagName                     = "GUID"
+	outputFormatFlag                 = "output-format"
+	outputFormatTextTemplatePathFlag = "text-output-template"
+	outputTypeJSON                   = "json"
+	outputTypeTEXT                   = "text"
+	outputTypeXML                    = "xml"
+	outputTypeInvalidMessage         = "Invalid output type '%s'. Please choose from: '%s', '%s', or '%s'."
```

Comment:
Should probably make this an error
```
var (
  ErrInvalidMessage = fmt.Errorf("Invalid output type. Please choose from json, text, xml")
)
```

That way users can check for this error directly.

Reply:
Don't indicate the type given?

Reply:
Well, I guess you could wrap the error with the type given if you wanted. 

This way, users can do something like `if err == cmd.ErrInvalidMessage {  }` (although since this IS the CLI it's kind of irrelevant...)

Reply:
Fair call